tests/scripts/setup_integration.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -x
+
+cd /home/circleci
+
+ls -al /var/www/html
+sudo rsync -a project/ /var/www/html/
+rm -rf project/*
+
+echo "Waiting for DB..."
+while ! mysql -uroot -h 127.0.0.1 -e "SELECT 1" >/dev/null 2>&1; do
+    sleep 1
+done
+
+mysql -u root -h 127.0.0.1 circle_test < magento-sample-data-1.9.2.4/magento_sample_data_for_1.9.2.4.sql
+php -d memroy_limit=512M n98-magerun.phar install --magentoVersionByName=magento-mirror-1.9.3.6 \
+  --installationFolder=/var/www/html --forceUseDb --noDownload --dbHost=127.0.0.1 --dbUser=root --dbName=circle_test  --dbPass="" \
+  --installSampleData=yes --useDefaultConfigParams=yes --locale=en_US --baseUrl=http://m1-test.integrations.dev.bolt.me
```

Comment:
@daisy1754 There seems to be a problem with the "--locale=en_US" option in circle CI
https://circleci.com/gh/BoltApp/bolt-magento1/5899?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link